SocketRocket/SRWebSocket.m

```diff
@@ -449,11 +471,8 @@ - (void)didConnect;
 - (void)_updateSecureStreamOptions
 {
     SRDebugLog(@"Setting up security for streams.");
-    [_securityOptions updateSecurityOptionsInStream:_inputStream];
-    [_securityOptions updateSecurityOptionsInStream:_outputStream];
-
-    SRDebugLog(@"Allows connection any root cert: %d", _securityOptions.validatesCertificateChain);
-    SRDebugLog(@"Pinned cert count: %d", _securityOptions.pinnedCertificates.count);
+    [_securityPolicy updateSecurityOptionsInStream:_inputStream];
```

Comment:
Failing tests and probably is going to fail to connect to `http` hosts with this.
Please wrap 474-475 in a `if (_requestRequiresSSL)` check.